### ISSUE TITLE
如果想搭建chatGLM2-6B本地模型，应该如何更改？

### ISSUE THREAD

**Issue by shanggangli** [OpenBMB/AgentVerse#24]

**Comment by JetSquirrel** [comment]
I'm interested in this, can you assign it to me?

**Comment by chenweize1998** [comment]
@JetSquirrel Certainly! We initially intended to use [Fastchat](https://github.com/lm-sys/FastChat) for integrating local models, but do not have time to  implement it. Fastchat offers an interface compatible to OpenAI's API for local models such as LLaMA and Vicunna, which might allow us to use our existing class for OpenAI models without needing new classes for local models. Could you first investigate Fastchat to see if it's compatible with ChatGLM and other Huggingface models?
